daemon/src/engine/sys/con_curses.cpp

```diff
@@ -272,6 +272,7 @@ CON_Resize
 The window has just been resized, move everything back into place
 ==================
 */
+// Here is guarded with _WIN32, later is guarded with SIGWINCH
```

Comment:
This is the only placed it's used:

https://github.com/mbasaglia/Unvanquished/blob/unused_functions/daemon/src/engine/sys/con_curses.cpp#L435-L438